### PR TITLE
Make Geomtry objects stringable

### PR DESCRIPTION
**Pull request by devinfd**
I needed a way to cast the geometry to a string.

### CODE REVIEW

File: src/Objects/Geometry.php [hunk 1]
```diff
@@ -23,6 +23,14 @@ abstract class Geometry implements Castable, Arrayable, Jsonable, JsonSerializab
   abstract public function toWkt(): string;
 
   abstract public function getWktData(): string;
+   
+ /**
+  * @return string
+  */ 
+  public function __toString()
```
Comment (MatanYadaev) [inline src/Objects/Geometry.php:30]:
```suggestion
  public function __toString(): string
```

File: src/Objects/Geometry.php [hunk 2]
```diff
@@ -23,6 +23,14 @@ abstract class Geometry implements Castable, Arrayable, Jsonable, JsonSerializab
...
+  {
+    return $this->getWktData();
```
Comment (MatanYadaev) [inline src/Objects/Geometry.php:32]:
```suggestion
    return $this->toWkt();
```

File: src/Objects/Geometry.php (hunk 1 above)
Comment (MatanYadaev) [inline src/Objects/Geometry.php:30]: Please add `implements Stringable` to this class

**Review by MatanYadaev — CHANGES_REQUESTED** [review]
Hi, thanks for the PR. I encourage you for next times to be more descriptive. Right now I have no idea why this PR is necessary. It would be great if you'll explain the motivation here.

File: src/Objects/Geometry.php (hunk 1 above)
Comment (MatanYadaev) [inline src/Objects/Geometry.php:30]: Can you please add unit tests for each geometry?

**Comment by devinfd** [comment]
Thanks for the feedback. Sorry for the poor description, I've been overly busy lately. In my particular use case I have a class that records changes to a models attributes. That class loops through each model attribute and coerces them to a string ie: `(string) $attribute`. Prior to this PR an error was thrown for any of the Object attributes.

Thanks for developing this Package!

**Review by MatanYadaev — APPROVED** [review]
Thanks!

**Pull request merged**